buildstockbatch/gcp/gcp.py

```diff
@@ -788,15 +791,94 @@ def process_results(self, skip_combine=False, use_dask_cluster=True):
+                else:
+                    postprocessing.combine_results(self.get_fs(), self.results_dir, self.cfg, do_timeseries=do_timeseries)
 
         finally:
             if use_dask_cluster:
                 self.cleanup_dask()
 
+
+    @classmethod
+    def run_combine_results_on_cloud(cls, gcs_bucket, gcs_prefix, results_dir, do_timeseries):
+        """This is the function that is run on the cloud to actually perform `combine_results` on
+        the cloud.
+        """
+        logger.info("run_combine_results_on_cloud starting")
+        client = storage.Client()
+        bucket = client.get_bucket(gcs_bucket)
+
+        logger.debug("Reading config")
+        blob = bucket.blob(f"{gcs_prefix}/config.json")
+        cfg = json.loads(blob.download_as_string())
+
+        DaskClient()
+        postprocessing.combine_results(GCSFileSystem(), results_dir, cfg, do_timeseries=do_timeseries)
+
+
+    def setup_combine_results_on_cloud(self, results_dir, do_timeseries=True):
+        """Set up `combine_results` to be run on GCP Cloud Run.
+
+        Parameters are passed to `combine_results` (so see that for parameter documentation).
+        """
+        logger.info("Creating job to run combine_results on Cloud Run...")
+
+        # Define the Job
+        pp_env_cfg = self.cfg["gcp"].get("postprocessing_environment", {})
+        job = run_v2.Job(
+            template=run_v2.ExecutionTemplate(
+                template=run_v2.TaskTemplate(
+                    containers=[
+                        run_v2.Container(
+                            name=self.job_identifier,
+                            image=self.repository_uri + ":" + self.job_identifier,
+                            resources=run_v2.ResourceRequirements(
+                                limits={
+                                    "memory": f"{pp_env_cfg.get('memory_mib', 4096)}Mi",
+                                    "cpu": str(pp_env_cfg.get('cpus', 2)),
+                                }
+                            ),
+                            command=["/bin/sh"],
+                            args=["-c", "python3 -m buildstockbatch.gcp.gcp"],
+                            env=[
+                                run_v2.EnvVar(name="JOB_TYPE", value="POSTPROCESS"),
+                                run_v2.EnvVar(name="GCS_PREFIX", value=self.gcs_prefix),
+                                run_v2.EnvVar(name="GCS_BUCKET", value=self.gcs_bucket),
+                                run_v2.EnvVar(name="RESULTS_DIR", value=results_dir),
+                                run_v2.EnvVar(name="DO_TIMESERIES", value="True" if do_timeseries else "False")
+                            ],
+                        )
+                    ],
+                    timeout=f"{60 * 60 * 24}s", # 24h
+                    max_retries=0,
+                )
+            )
+        )
+
+        # Create the job
```

Comment:
Can you add a TODO to `clean()` to clean up the Cloud Run job? (Or add that cleanup now.)

Reply:
I added the cleanup now!